### DISCUSSION TITLE
protoc.exe path

### DISCUSSION THREAD

**Discussion by SemihSaydamAndroid** [zalopay-oss/jmeter-grpc-request#65] [Q&A]
Hello,

I fill in all the given fields correctly. But when I click on "listing" I get the following error:

...java.lang.RuntimeException: Unable to resolve service by invoking protoc
..Caused by: java.io.IOException: Cannot run program "C:\Users\USERNAME\protocjar6732940451156264637\bin\protoc.exe": CreateProcess error=1260, This program is blocked by group policy. For more information, contact your system administrator

Is it possible to run protoc.exe via a different file? I don't have permission to run jar on the path shown in the error.

Kind regards.

**Comment by yl-yue** [comment]
Please log in to your computer with an account with higher authority
